### PR TITLE
Docs e2e enhanced

### PR DESCRIPTION
**Pull request by manuvaldi**

### CODE REVIEW

**Comment by jtudelag** [comment]
@manuvaldi are you missing content here? I can't see anything related with E2E...

**Comment by manuvaldi** [comment]
> @manuvaldi are you missing content here? I can't see anything related with E2E..
I forgot the commit :)

**Review by ernesgonzalez33 — APPROVED** [review]
LGTM

File: documentation/E2E_TESTING.md [hunk 1]
```diff
@@ -1,17 +1,28 @@
 # E2E Testing
 
+This End-2-End tests have been developed using Selenium. Selenium is a open-source automated testing framework used to validate web applications across different browsers and platforms. You can use multiple programming languages like create Selenium Test Scripts.
```
Comment (jtudelag) [inline documentation/E2E_TESTING.md:3]: Add link to selenium documention.

File: documentation/E2E_TESTING.md [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
 
 ```shell
 cd tests/selenium-grid
 helm install --wait <my-selenium-release> . -n <PLATFORM_NAMESPACE>
 ```
 
-After this, you can run the e2e test the following way:
+### 2. Change `values.yaml`
+
+Change `values.yaml` file to adjust it to your deployment. (The most variables are fine to a default deployment)
```
Comment (jtudelag) [inline documentation/E2E_TESTING.md:21]: This sentence "(The most variables are fine to a default deployment)" is a bit confusing in English...can we rewrite it?

Comment (jtudelag) [inline documentation/E2E_TESTING.md:21]: which values.yaml? From selenium chart? From the e2e chart¿

File: documentation/E2E_TESTING.md (hunk 1 above)
Comment (manuvaldi) [reply documentation/E2E_TESTING.md]: take a look now @jtudelag, please

File: documentation/E2E_TESTING.md (hunk 2 above)
Comment (manuvaldi) [reply documentation/E2E_TESTING.md]: take a look now @jtudelag, please

Comment (manuvaldi) [reply documentation/E2E_TESTING.md]: take a look now @jtudelag, please

**Pull request merged**